example/routeguide/BUILD.bazel

```diff
@@ -129,18 +129,54 @@ proto_compile(
 
 proto_compile(
     name = "routeguide_go_compile",
-    output_mappings = ["routeguide.pb.go=github.com/stackb/rules_proto/example/routeguide/routeguide.pb.go"],
-    outputs = ["routeguide.pb.go"],
-    plugins = ["@build_stack_rules_proto//plugin/golang/protobuf:protoc-gen-go"],
+    options = {
+        "@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway:protoc-gen-grpc-gateway": [
+            "generate_unbound_methods=true",
+            "logtostderr=true",
+        ],
+        "@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway:protoc-gen-grpc-openapiv2": [
+            "logtostderr=true",
+            "use_go_templates=true",
+        ],
+    },
+    output_mappings = [
+        "routeguide.pb.go=github.com/stackb/rules_proto/example/routeguide/routeguide.pb.go",
+        "routeguide_grpc.pb.go=github.com/stackb/rules_proto/example/routeguide/routeguide_grpc.pb.go",
+    ],
+    outputs = [
+        "routeguide.pb.go",
+        "routeguide.pb.gw.go",
+        "routeguide.swagger.json",
+        "routeguide_grpc.pb.go",
+    ],
+    plugins = [
+        "@build_stack_rules_proto//plugin/golang/protobuf:protoc-gen-go",
+        "@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway:protoc-gen-grpc-gateway",
+        "@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway:protoc-gen-grpc-openapiv2",
+        "@build_stack_rules_proto//plugin/grpc/grpc-go:protoc-gen-go-grpc",
+    ],
     proto = "routeguide_proto",
 )
 
 proto_go_library(
     name = "routeguide_go_proto",
-    srcs = ["routeguide.pb.go"],
+    srcs = [
```

Comment:
A separate example is probably warranted for grpc-gateway rather than adding it to routeguide

Reply:
I'll try to provide a separate /golden/testdata/grpc-gateway example as well as a working (well, maybe not really, as the routeguide_go_proto build doesn't work) golang example.